src/System.Diagnostics.StackTrace.Symbols/src/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.Platforms": "1.0.1-rc3-24112-00",
+    "System.Collections": "4.0.11-rc3-24112-00",
+    "System.Collections.Immutable": "1.2.0-rc3-24112-00",
+    "System.Diagnostics.Debug": "4.0.11-rc3-24112-00",
+    "System.IO": "4.1.0-rc3-24112-00",
+    "System.IO.FileSystem": "4.0.1-rc3-24112-00",
+    "System.Reflection.Metadata": "1.3.0-rc3-24112-00",
+    "System.Runtime": "4.1.0-rc3-24112-00",
+    "System.Runtime.Extensions": "4.1.0-rc3-24112-00",
+    "System.Runtime.InteropServices": "4.1.0-rc3-24112-00"
```

Comment:
As of what's in the repo right now, all of these 24112 values should instead be 24113.  Hopefully later today the repo will be updated to 24117, at which point if this hasn't been merged these will need to be updated to that value in this PR, too.